crates/plugins/mediator-coordination/src/web/coord/handler/stateful.rs

```diff
@@ -337,16 +337,18 @@ pub async fn process_plain_keylist_update_message(
 
     let mediator_did = &state.diddoc.id;
 
-    Ok(Message::build(
-        format!("urn:uuid:{}", Uuid::new_v4()),
-        KEYLIST_UPDATE_RESPONSE_2_0.to_string(),
-        json!(KeylistUpdateResponseBody {
-            updated: confirmations
-        }),
+    Ok(
+        Message::build(
+            format!("urn:uuid:{}", Uuid::new_v4()),
```

Comment:
There's a more transparent method for this.
```suggestion
            Uuid::new_v4().urn().to_string(),
```
What is the purpose of the **urn:uuid:** prefix?
cc @IngridPuppet

Reply:
there is not purpose just wanting to give the id a format except you planned using